library(testthat)
library(exception)

test_check("exception")
